evil-occur.el

```diff
@@ -43,10 +43,10 @@
     [mouse-2] 'occur-mode-mouse-goto
     (kbd "C-c C-c") 'occur-mode-goto-occurrence
 
-    ;; goto
-    (kbd "C-m") 'occur-mode-goto-occurrence
-    "o" 'occur-mode-goto-occurrence-other-window
-    (kbd "C-o") 'occur-mode-display-occurrence
+    ;; follow
+    (kbd "<return>") 'occur-mode-goto-occurrence
+    (kbd "S-<return>") 'occur-mode-goto-occurrence-other-window
```

Comment:
Same here, story for terminal usage.

(Last comment on this even though there are multiple files which would want one.)

Reply:
Can be set to `o`/`O`, there is no conflict.